kong-3.0.1-0.rockspec

```diff
@@ -1,10 +1,10 @@
 package = "kong"
-version = "3.3.0-0"
+version = "3.0.1-0"
```

Comment:
The filename and this change seem wrong.

Reply:
It is part of our changelist. After we release, we bump the version numbers. We do the same in master (when we made feature freeze, we bumped the version number in master to 3.4)

Reply:
oh but you are right, it should be 3.3.1, not 3.0.1. I must have tripped over the keyboard, fixing.

Reply:
Should be good now